src/components/Grid/Position.js

```diff
@@ -42,8 +42,9 @@ export const Position = (props) => {
     let x
 
     const y = props.refs.findIndex((columnRefs) => {
-      x = columnRefs.findIndex((columnRef) =>
-        columnRef.current.contains(event.target)
+      x = columnRefs.findIndex(
+        (columnRef) =>
+          columnRef.current && columnRef.current.contains(event.target)
```

Comment:
This fixes `Fix sort column header console error` discovered in Nora

Reply:
![image](https://user-images.githubusercontent.com/142403/73022735-ffb04e80-3dde-11ea-9ae9-5e2b698356a2.png)

above is original error. Here's after no error after clicking just outside of the sort field (not clicking directly on the sort icons/anchor was the repro):

![image](https://user-images.githubusercontent.com/142403/73022804-21113a80-3ddf-11ea-81ff-82ac9956f6af.png)